src/server.h

```diff
@@ -41,6 +41,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <map>
 #include <vector>
 
+bool disable_asciiart(bool,bool);
```

Comment:
Could you please replace that line with `bool disable_asciiart(bool value, bool flag)`? (variable names + space between the arguments).